docs/reference/services.rst

```diff
@@ -0,0 +1,63 @@
+=================
+MicroOVN Services
+=================
+
+This document provides brief explanation of services provided by MicroOVN. They
+are generally not meant for direct user interaction and should be used only
+if necessary. List and status of these services can be displayed by running:
+
+.. code-block:: none
+
+    snap services microovn
```

Comment:
Align the first letter of the command ('s') directly under the 'c' (of code-block). This should be done everywhere in the document, even for 'note' or 'warning', etc.